app/helpers/email_helper.rb

```diff
@@ -28,4 +27,12 @@ def whitehall_base_email_sign_up_url
   def equivalent_whitehall_url
     root + request.fullpath
   end
+
+  def is_accordion?(presented_taxon)
```

Comment:
The `TaxonPresenter` class has the `renders_as_accordion?` method, we could use that instead?